src/probnum/__init__.py

```diff
@@ -23,7 +25,8 @@
 
 from pkg_resources import DistributionNotFound, get_distribution
 
-from . import diffeq, filtsmooth, linalg, linops, quad, random_variables, utils
+from . import diffeq, filtsmooth, kernels, linalg, linops, quad, random_variables, utils
+from .kernels import Kernel
```

Comment:
what is the reason for this extra import?

Reply:
I removed the Kernel import. This should not be at top-level, I agree.